.github/workflows/update_dependencies_zulip.yml

```diff
@@ -40,7 +40,7 @@ jobs:
                 });
               }
             } else {
-              output += "No PR found for this run!";
+              output += "No PR found for this run! Please run `lake update` on master and make a PR.";
```

Comment:
Since we have an automated job that makes PRs every hour or so, are we sure that we want to ask people to open PRs manually?

I haven't examined the timing of when this message has been posted, but is it possible that the PR associated to the commit was closed before the job finished or something like that?

Perhaps for debugging purposes we should just have the bot spit out `context.payload` or something so we can see why this was triggered?

Reply:
Ah, the update dependencies workflow has a manual trigger, so we could also say "if you have write access, [go to this page](https://github.com/leanprover-community/mathlib4/actions/workflows/update_dependencies.yml) and click the "Run workflow" button".